src/ui/ErrorReport/ErrorReport.ts

```diff
@@ -247,6 +247,12 @@ export class ErrorReport extends Component {
         this.setErrorTitle(l('CannotAccess', this.organizationId), l('InvalidToken'));
         break;
 
+      case 'GroupByAndFacetBothExistingException':
+        this.options.showDetailedError = false;
+        // TODO: add MLFacet documentation
+        this.setErrorTitle(undefined, l('CannotUseBothFacetTypes'));
```

Comment:
Suggestion:

errorName could be something along those lines: `GroupByAndFacetRequestsCannotCoexist`
helpSuggestion string could be: `The query is invalid because it contains both Group By and Facet requests. Ensure that the search interface does not initialize MLFacet components alongside Facet components, or any component extending the Facet component, such as FacetRange or FacetSlider).`

Reply:
Great!